src/style.css

```diff
@@ -0,0 +1,85 @@
+body {
+  margin: 0;
+}
+
+.class {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.box {
+  height: 300px;
+  min-width: 300px;
+  flex-grow: 1;
+
+  font-family: Arial, Helvetica, sans-serif;
+  color: white;
+  font-size: 100px;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.box1 {
+  background-color: rgb(100%, 0%, 0%);
+}
+
+.box2 {
+  background-color: rgb(80%, 0%, 0%);
+}
+
+.box3 {
+  background-color: rgb(60%, 0%, 0%);
+}
+
+.box4 {
+  background-color: rgb(40%, 0%, 0%);
+}
+
+.box5 {
+  background-color: rgb(20%, 0%, 0%);
+}
+
+.box6 {
+  background-color: rgb(0, 0, 0);
+}
+@media (min-width: 600px) {
+  .class {
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
```

Comment:
make that a default values for your container, move them outside of media rule

Reply:
done - but I just did not transfer  flex-direction: row to default, because then you need to create media for the column with a length of less than 300 px. Could I left it like it is now?